test_libs/pyspec/eth2spec/fuzzing/decoder.py

```diff
@@ -0,0 +1,84 @@
+from eth2spec.utils.ssz import ssz_typing as spec_ssz
+import ssz
+
+
+def translate_typ(typ) -> ssz.BaseSedes:
```

Comment:
Do we want to add some tests here?
Because we don't use this anywhere in the spec-dev, CI, or generators, the chances of missing a corresponding update here when we change SSZ is high. For example, adding max length to lists.

Reply:
Agree, will transform the example usage in a test that goes over all phase-0 types.

Reply:
Included a test but not sure where I like the test to be located. Kept it near the code for now, since it's not really a spec test.

Reply:
seems appropriate for now 👍